views/partials/social_media_sharing_buttons.hbs

```diff
@@ -1,10 +1,12 @@
 <ul class="social-media-sharing-buttons">
   <li>
+    <a href="https://www.facebook.com/sharer/sharer.php?u={{ SERVER_URL}}" target="_blank" rel="noopener noreferrer">
```

Comment:
`rel="noopener noreferrer"` will automatically cause the browser to open the link in a new tab/window. No need to specify `target="_blank"`